hack/tests/conformance-gce.sh

```diff
@@ -32,6 +32,8 @@ BOOTKUBE_VERSION=${BOOTKUBE_VERSION:-}
 COREOS_CHANNEL=${COREOS_CHANNEL:-'coreos-stable'}
 WORKER_COUNT=4
 GCE_PREFIX=${GCE_PREFIX:-'bootkube-ci'}
+GCE_ACCOUNT=${GCE_ACCOUNT:-'bootkube-ci'}
```

Comment:
Just preserving what was here. 

It appears that we have a service account titled: `bootkube-ci@coreos-gce-testing.iam.gserviceaccount.com` That we don't want to change with the instance prefix change because it refers to a specific account. The default instance prefix just happened to be the same as the service account but if we actually want to chance the instance prefix we shouldn't be changing the account as well.